### ISSUE TITLE
If you have letsencrypt have copy the keys your self

### ISSUE THREAD

**Issue by RaymondDay61** [cockpit-project/cockpit#11230]
Page:

**Comment by RaymondDay61** [comment]
Wow I spent all day trying to get the Cockpit to work. Going to port 9090 just could not be displayed.

When I did a apt install cockpit it took a long time and got errors. I guess because I all ready have Letsencrypt on my server.

The fix was to copy my letsencrypt keys to one file named 1-my-cert.cert to /etc/cockpit/ws-certs.d.

So in that file was 2 end and begin key's. certificate, and private key. I took them from the letsencrypt folder at /etc/letsencrypt/live/myserver.com/

Why can't the cockpit install look for this and do all that.

When I did a apt install cockpit on another Ubuntu server I have but don't have letsencrypt on it. It was fast and the port 9090 came right up. So I know it was something wrong with my other server with letsencrypt installed.

-Raymond Day
